FrejaEidClient/src/main/java/com/verisec/frejaeid/client/beans/general/OrganisationIdInfo.java

```diff
@@ -0,0 +1,71 @@
+package com.verisec.frejaeid.client.beans.general;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * Contains information regarding organisation ID and its issuer.
+ */
+public class OrganisationIdInfo {
+
+    private final String identifier;
+    private final Map<String, String> issuerFriendlyName;
+    private final String issuerCode;
+
+    /**
+     * Creates instance of {@linkplain OrganisationIdInfo}.
+     *
+     * @param identifier         User's Organisation ID. It cannot be {@code null} or empty.
+     * @param issuerFriendlyName Friendly name of the organisation which issued the user's Organisation ID
+     *                           in Swedish and English. It cannot be {@code null} or empty.
+     * @param issuerCode         Unique identifier of the organisation which issued the user's Organisation ID. It
```

Comment:
This one can be empty (and will be probably for all that does not share their org id with sweden connect Rps)